Part 2

```diff
@@ -0,0 +1,13 @@
+Matthew Danielson
+6/30/16
+Answers to questions for Part 2 of Assignment 3
+
+1) It is not valid; for one, variable i is not declared; we would have to assume that it has been declared already. The printf statement is also incrrect, as nothing is actually being printed, which will cause an error.
```

Comment:
we'd actually have to assume that it was initialized as well - and it must be initialized to something greater than 0 to properly run.